### PR TITLE
Added a caution note about the LDAP injection attacks

### PR DESCRIPTION
**Pull request by javiereguiluz**
After reading [this comment](https://github.com/symfony/symfony-docs/issues/6795#issuecomment-248075845) by @csarrazi I'm not sure which protection does the LDAP component offer and which one it doesn't ... so please, review this carefully. Thanks!

### CODE REVIEW

**Comment by csarrazi** [comment]
👍

**Review by HeahDude — APPROVED** [review]

File: security/ldap.rst [hunk 1]
```diff
@@ -167,6 +167,13 @@ use the ``ldap`` user provider.
             ),
         );
 
+.. caution::
+
+    The Security component escapes values provided when binding against an LDAP
+    server (likewise for the user provider). However, the LDAP component does
+    not provide any other escaping, so it's your responsibility to prevent
+    the LDAP injection attacks.
```
Comment (xabbuh) [inline security/ldap.rst]: I would reword this a bit:

> The Security component escapes provided input data when the LDAP user provider is used. However, the LDAP component itself does not provide any escaping yet. Thus, it's your responsibility to prevent LDAP injection attacks when using the component directly.

@javiereguiluz @csarrazi What do you think?

Comment (javiereguiluz) [inline security/ldap.rst]: @xabbuh your version is much better because it's easier to understand! Thanks.

**Comment by xabbuh** [comment]
Thank you Javier.

File: security/ldap.rst (hunk 1 above)
Comment (csarrazi) [inline security/ldap.rst]: I also agree
